tuning/tune.py

```diff
@@ -47,16 +48,76 @@ class OpWalkResult:
     dispatch_kind: DispatchKind = None
 
 
+class ElementType(Enum):
+    i8 = 1
+    i32 = 2
+    f8 = 3
+    f16 = 4
+    f32 = 5
+
+    @property
+    def bitwidth(self) -> int:
+        return {
+            ElementType.i8: 8,
+            ElementType.i32: 32,
+            ElementType.f8: 8,
+            ElementType.f16: 16,
+            ElementType.f32: 32,
+        }[self]
```

Comment:
```suggestion
    _bitwidth_map = {
        'i8': 8,
        'i32': 32,
        'f8': 8,
        'f16': 16,
        'f32': 32,
    }

    @property
    def bitwidth(self) -> int:
        return self._bitwidth_map[self.name]
```

Reply:
Why would we prefer this?

Reply:
this way it defines the mapping as a class attribute, simplifies the `bitwidth` method. Both ways work

Reply:
I don't see how it simplifies the method. The biggest difference to me is that it introduces a new attribute, and I'd rather keep this as an implementation detail. I'm going to keep this as-is.